### PR TITLE
Add classifier_activation param to applications

### PR DESCRIPTION
**Pull request by ivallesp**
I have noticed that the DenseNet and NasNet models did not have the possibility to specify the `classifier_activation` parameter, when the main classes have it implemented and the other models in `tf.keras.applications` also have them available, hence I have added those parameters. I have tested the code locally.

### CODE REVIEW

**Review by fchollet — CHANGES_REQUESTED** [review]
Thank you for the PR! Please add a unit test to check this change. Also, you may need to update the API pbtxt files since this is an API change.

**Comment by ivallesp** [comment]
I have added the unit tests and squashed the commit with the previous one. Could you please elaborate on the pbtxt request? What do you need me to change?

**Review by fchollet — APPROVED** [review]
LGTM, thank you

**Comment by fchollet** [comment]
> Could you please elaborate on the pbtxt request? What do you need me to change and where?

I've checked that we should be able to handle that on our side.

**Comment by rthadur** [comment]
@ivallesp can you please check sanity build failures ?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!
